### PR TITLE
atftpd: update init script to use procd and publish service

### PR DESCRIPTION
**Pull request by mhusaam**
* update init script to use procd
* announce tftp service over mdns

Signed-off-by: Mohd Husaam Mehdi <husaam.mehdi@iopsys.eu>

### CODE REVIEW

**Comment by mhusaam** [comment]
Done.

File: net/atftp/files/atftpd.init [hunk 1]
```diff
@@ -18,9 +19,11 @@ start_service() {
 
 	config_get srv service path "/srv/tftp"
 	config_get port service port 69
+	config_get mdns service mdns 1
```
Comment (feckert) [inline net/atftp/files/atftpd.init:22]: Please do not set mdns to `1` what means `enabled`.
The default behavior should be `0` therefore `disabled`.

The behavior should not change

**Review by feckert — CHANGES_REQUESTED** [review]
Sorry for the late repley. See my comments.

File: net/atftp/files/atftpd.init (hunk 1 above)
Comment (mhusaam) [reply net/atftp/files/atftpd.init]: Hello @feckert,

Apologies for the very late response.

Set to 0 by default.

Kindly review.

**Comment by mhusaam** [comment]
updated, kindly review

**Comment by feckert** [comment]
Thanks merged!

**Comment by mhusaam** [comment]
Hi @feckert, thanks a lot.

Could you also please take a look at: https://github.com/openwrt/packages/pull/23752?

**Pull request merged**